mailbox/jpa/src/main/java/org/apache/james/mailbox/jpa/JPAMailboxManager.java

```diff
@@ -41,9 +42,9 @@
 public abstract class JPAMailboxManager extends StoreMailboxManager {
     
     public JPAMailboxManager(JPAMailboxSessionMapperFactory mailboxSessionMapperFactory,
-            final Authenticator authenticator, MailboxPathLocker locker, MailboxACLResolver aclResolver, 
```

Comment:
because we think final flag for argument are unnecessary

The signature becomes harder to read, and we are anti-reallocation nazis ^^